latex/solution.tex

```diff
@@ -1,13 +1,13 @@
 \documentclass[rgb,dvipsnames,aspectratio=169,9pt,t]{beamer}
 \input{solutions-base.tex}
 \begin{document}
-\begingroup\graphicspath{{{%problemdir%}/problem_statement/}}
+\begingroup\graphicspath{{{%problemdir%}/solution/}{{%problemdir%}/statement/}}
```

Comment:
It is so nice that this just works :heart_eyes: Tested this on `nwerc2024/practice/blackboard`, both with the `state*.jpg`s in `statement/` and in `solution/`, and both compile just fine :relaxed:

Reply:
:D